group_vars/donbot_azure/vars.yml

```diff
@@ -0,0 +1,67 @@
+---
+stack_prefix: 'don'
```

Comment:
```suggestion
stack_prefix: 'db'
```

Reply:
If we don't **have** to have 2 letter prefix, then I would prefer having `don` instead of `db`.

Reply:
We have several prefixes longer than 2 letters, but I'll go along and make it more compatible. Accepted